pandas/tests/arrays/interval/test_interval.py

```diff
@@ -93,6 +113,221 @@ def test_set_na(self, left_right_dtypes):
         tm.assert_extension_array_equal(result, expected)
 
 
+class TestComparison:
```

Comment:
possibly put in tests.arithmetic.test_interval and parametrize with box_with_array